web/client/epics/mapexport.js

```diff
@@ -59,6 +59,7 @@ const PersistMap = {
         }
 
         return Rx.Observable.forkJoin(...layers.map(layer => getLayerCapabilities(layer).catch(() => Rx.Observable.of(null))))
+            .switchMap(capArr => capArr.map(({ style, ...cap}) => cap))
```

Comment:
Hi @alexander-fedorenko , I see this is an export for WMC format and I'm not sure the issue wants to target that kind of export. I think the WMC export should be kept as is now.
@tdipisa could you please confirm this?

Reply:
WMC Export is not involved at all here.

Reply:
@alexander-fedorenko please remove the fix you introduced for the WMC export. The other files seem good

Reply:
@allyoucanmap @tdipisa Changes in WMC export are rolled-back now. PR description is updated as well.